README.md

```diff
@@ -1,57 +1,180 @@
+
+- Country: Select the desired country from the drop-down menu.
+- Number or Location: Select the desired option to search by digits/phrases, or a specific City or Region.
+- Capabilities: Select your service needs for this number. 
+
+3. Click Buy to purchase a phone number for your current project or sub-account.
+![Search Results](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/resources/search-phone-number.png?raw=true)
+> **Notice:** Many countries require identity documentation for Phone Number compliance. Requests to provision phone numbers with these regulations will be required to select or add the required documentation after clicking Buy in Console. To see which countries and phone number types are affected by these requirements, please see twilio's [Phone Number Regulations](https://www.twilio.com/guidelines/regulatory) site.
+
+### Step 3: Obtain a Twilio Account SID with Auth Token.
+Twilio uses two credentials to determine which account an API request is coming from: The Account SID, which acts as a `username`, and the Auth Token which acts as a `password`. You can find your account SID and auth token in your [Twilio console](https://www.twilio.com/console).
+
+![Twilio Credentails](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/resources/get-credentails.png?raw=true)
+
+Your account's Auth Token is hidden by default. Click show to display the token, and hide to conceal it again. For further information click [here](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them)
+
+## Sample
+
+This sample demonstrates a scenario where the Twilio connector is used to send a text message to a phone number.
+
+### Step 1 - Import the package
+
+Import the Twilio package into your Ballerina program as shown below:
+
+```ballerina
+import ballerinax/twilio;
+```
+
+### Step 2 - Create a new connector instance
+
+To create a new connector instance, add a configuration as follows (You can use [configurable variables](https://ballerina.io/learn/by-example/configurable.html) to provide the necessary credentials):
+
+```ballerina
+configurable string accountSID= ?;
+configurable string authToken = ?;
+
+twilio:ConnectionConfig twilioConfig = {
+    auth: {
+        username: accountSID,
+        password: authToken
+    }
+};
+
+twilio:Client twilio = check new (twilioConfig);
+```
+
+### Step 3 - Invoke the connector operation
+
+1. Invoke the connector operation using the client as shown below:
+
+```ballerina
+public function main() returns error? {
+    twilio:Client twilio = check new (twilioConfig);
+
+    twilio:CreateMessageRequest messageRequest = {
+        To: "+XXXXXXXXXXX",
+        From: "+XXXXXXXXXXX",
+        Body: "Hello from Ballerina"
+    };
+
+    twilio:Message response = check twilio->createMessage(accountSID, messageRequest);
+    
+    io:print(response?.status);
```

Comment:
```suggestion

    twilio:Message response = check twilio->createMessage(accountSID, messageRequest);
    io:print(response?.status);
```

Reply:
since this is the primary example we are providing to the user, IMO we better provide some meaningful output, instead of just printing the boolean status.

Reply:
Yes of course. but here the status mean the status of the message after we send it. It can be,
- queued
- sending
- sent
- failed
- delivered
- undelivered
- receiving
- received
- accepted
- scheduled
- read
- partially_delivered
- canceled

Reply:
Further, I added extra description in [5d64220](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/5d64220725ae541b41890724532148c1a2154da8).